spec/system/placements/schools/mentors/find_a_mentor_with_no_results_spec.rb

```diff
@@ -0,0 +1,182 @@
+require "rails_helper"
+
+RSpec.describe "School user searches for a mentor with no results", service: :placements, type: :system do
+  before do
+    stub_no_results_teaching_record_response
+  end
+
+  scenario do
+    given_mentor_and_school_exists
+    and_i_am_signed_in
+
+    when_i_am_on_the_mentors_index_page
+    and_i_click_on_add_mentor
+    then_i_see_the_find_mentor_page
+
+    when_i_click_on_help_with_the_trn
+    then_i_see_link_to_trn_guidance
+
+    # user does not enter any data
+    when_i_click_on_continue
+    then_i_see_a_validation_error_for_not_entering_any_data
+
+    # user only enters date of birth
+    when_i_enter_a_date_of_birth
+    and_i_click_on_continue
+    then_i_see_a_validation_error_for_not_entering_a_trn
+
+    # user only enters trn
+    when_i_clear_the_date_of_birth
+    and_i_enter_a_trn
+    and_i_click_on_continue
+    then_i_see_a_validation_error_for_not_entering_a_date_of_birth
+
+    when_i_enter_an_invalid_trn
+    and_i_click_on_continue
+    then_i_see_a_validation_error_for_not_entering_a_valid_trn
+
+    when_i_enter_valid_data_that_does_not_exist_in_the_teaching_record_service
+    and_i_click_on_continue
+    then_i_see_the_no_results_page
+
+    when_i_click_on_change_your_search
+    then_i_see_the_find_mentor_form_with_the_trn_and_date_of_birth_prefilled
+  end
+
+  private
+
+  def given_mentor_and_school_exists
+    @school = build(:school, :placements, name: "Springfield Elementary")
+    @new_mentor = build(:placements_mentor, first_name: "Mister", last_name: "Bergstrom")
```

Comment:
This mentor isn't current attached to the school you're testing, given you're using `"8888888"` elsewhere, it may be worth removing this mentor and replacing the trn in the `and_i_enter_a_trn` with `"8888888"` instead?